docs/shop4cfdefinitions.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$schemaVersion": "0.0.1",
+    "$id": "https://smart-data-models.github.io/dataModel.SmartManufacturing/SHOP4CF/schema.json",
```

Comment:
Wouldn't it be better a correct URL to this `shop4cfdefinitions.jsonld` file?
i.e. `https://raw.githubusercontent.com/shop4cf/data-models/master/docs/shop4cfdefinitions.jsonld` ?

Other projects may also claim to define the 'official' SmartManufacturing schema. I would use our URLs, before our schema gets accepted as the official one.

Reply:
Yes, we should move to a one self-hosted. This was just as test